src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js

```diff
@@ -729,9 +734,9 @@
             clearNotifications($scope.content);
             // TODO: Add "..." to save button label if there are more than one variant to publish - currently it just adds the elipses if there's more than 1 variant
             if (hasVariants($scope.content)) {
-                //before we launch the dialog we want to execute all client side validations first
-                if (formHelper.submitForm({ scope: $scope, action: "openSaveDialog" })) {
 
+                //before we launch the dialog we want to execute all client side validations first
```

Comment:
Is this comment correct still, if you are skipping client validation?!?

Reply:
Well, we still launch them, we just don't allow them to be blocking. And still this is only affected by save, not save & publish.